dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task220413IncreasePublishedPushedAssetIdColTest.java

```diff
@@ -0,0 +1,102 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.sql.SQLException;
+import java.util.Date;
+import java.util.UUID;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class Task220413IncreasePublishedPushedAssetIdColTest {
+
+    private static final String ASSET_ID = "this-is-an-assetid-which-is-longer-than-expected";
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    /**
+     * Tests Task220413IncreasePublishedPushedAssetIdCol by inserting to published_pushed_assets with an assetId with
+     * greater capacity that it can hold.
+     * An exception is expected to be thrown.
+     *
+     * @throws DotDataException
+     */
+    @Test
+    public void test_upgradeTask_noCapacity() throws DotDataException {
+        rollbackToOriginal();
+
+        try {
+           insertPublishedAsset(ASSET_ID);
+        } catch (DotDataException e) {
```

Comment:
You can do `@Test(expected = DotDataException.class)`

Reply:
@dsilvam feedback applied.